cmd/proxy/config/config.go

```diff
@@ -8,10 +8,19 @@ import (
 
 // Config holds the config for covalent proxy
 type Config struct {
-	Port              uint32 `toml:"port"`
-	HyperBlockPath    string `toml:"hyperBlockPath"`
-	ElrondProxyUrl    string `toml:"elrondProxyUrl"`
-	RequestTimeOutSec uint64 `toml:"requestTimeOutSec"`
+	Port                   uint32                 `toml:"port"`
+	HyperBlockPath         string                 `toml:"hyperBlockPath"`
+	ElrondProxyUrl         string                 `toml:"elrondProxyUrl"`
+	RequestTimeOutSec      uint64                 `toml:"requestTimeOutSec"`
+	HyperBlockQueryOptions HyperBlockQueryOptions `toml:"hyperBlockQueryOptions"`
+}
+
+type HyperBlockQueryOptions struct {
+	WithLogs            bool   `toml:"withLogs"`
+	WithAlteredAccounts bool   `toml:"withLogs"`
+	NotarizedAtSource   bool   `toml:"notarizedAtSource"`
+	Tokens              string `toml:"tokens"`
```

Comment:
I think this configuration holder file can be held under `covalent-indexer-go/config` directory, in accordance with other projects

Reply:
Used the same pattern from `tools` repository. Also, this config is specific to the loaded config, so I would leave it as it is